src/commands/social.ts

```diff
@@ -50,6 +51,11 @@ export default {
           'LinkedIn do IFPE Open Source:\nhttps://www.linkedin.com/company/ifpe-open-source',
         );
         break;
+      case 'discord':
+        await interaction.reply(
+          'Servido no Discord do IFPE Open Source: \nhttps://discord.gg/6RdDJWQM',
```

Comment:
O link utilizado expira em 5 dias. Na sugestão, coloquei um convite que não tem prazo.

```suggestion
          'Servido no Discord do IFPE Open Source: \nhttps://discord.gg/QjvN9HRA6C',
```